# Fix validation errors in `pyproject.toml` on Python 3.13+ (strict PEP 621).
# 1. Updates `license` from string to table format.
# 2. Removes `license-files` (invalid key in [project] table).
# Author: Emik Lin (HKUMed CPOS)
--- pyproject.toml.orig	2025-09-24 09:01:48.000000000 +0800
+++ pyproject.toml	2026-01-22 11:01:44.437117227 +0800
@@ -13,8 +13,7 @@
 description = "Python interface to the R language (embedded R)"
 readme = "README.md"
 requires-python = ">=3.8"
-license = "GPL-2.0-or-later"
-license-files = ["LICENSE"]
+license = {text = "GPL-2.0-or-later"}
 authors = [{ name = "Laurent Gautier", email = "lgautier@gmail.com" }]
 classifiers = [
     "Programming Language :: Python",
